rest/route/patch.go

```diff
@@ -581,7 +581,7 @@ func makeSchedulePatchHandler(env evergreen.Environment) gimlet.RouteHandler {
 // Factory creates an instance of the handler.
 //
 //	@Summary		Configure/schedule a patch
-//	@Description	Update the list of tasks that the specified patch will run. This works both for initially specifying a patch's tasks, as well as for adding additional tasks to an already-scheduled patch.
+//	@Description	Update the list of tasks that the specified patch will run. This will only add on tasks, existing tasks will not be removed. This works both for initially specifying a patch's tasks, as well as for adding additional tasks to an already-scheduled patch.
```

Comment:
Instead of changing the routes default behavior, should we just add a parameter that tells the route to keep existing tasks? How would a user remove a task via the api with this change? Also do you know if this affects the UI re-configuring a task?

Reply:
Removing tasks already doesn't work. If you schedule a patch with 5 tasks, click reconfigure, remove four, it will still have 5. 
The same route is used for the UI. However, this only affects repeat patch.